SEP-0002.md

```diff
@@ -3,203 +3,288 @@
-and can be re-elected. It is recommended that the ED choose a deputy ED.
-
-* **Advise and Consent**: The ED shall present the state of the project and
-future plans at meetings of the board. The advisory board shall advise and
-consent on those plans.
-
-### The Executive
-
-The role of the executive which is headed by the Executive Director, is to
-manage the day to day operations of the SunPy organization. The executive shall:
-
-* hold regular meetings with the developer community (at least monthly)
-* make regular reports to the board (at least quarterly)
-* have push privileges to the SunPy repository and can delegate those privileges
-* have ownership privileges to the SunPy GitHub organization
-* create and maintain the SunPy code of conduct (see below)
-* shall plan and manage finances
-
-The current Executive Director is named in SEP-0006 and the structure of the
-executive is defined in SEP-0010.
-
-#### Code of Conduct
-
-The SunPy organization shall have an explicit code of conduct to state
-the norms of peer interactions, including inclusiveness and respect,
-and describe the process for dispute resolution.
-The code of conduct shall be publicly posted on the SunPy website.
-The code of conduct shall be accessible in such
-a manner as to welcome and incorporate feedback from the developer
-community, even anonymous feedback.
+The primary functions of the Advisory Board are:
 
-### The Advisory Board
+* to advise the Steering Committee,
+* aid in finding funding for the project to ensure the core aims of the project can be met,
+* nominate a Ombudsperson,
+* ensure that the Project is being run consistent with the approved governance structure and the needs of the Community.
 
-The board or executive director may appoint an advisory board to
-advise the board and executive director.
+The Advisory Board is self-electing, with members appointed by a vote of the existing Advisory Board.
 
-## Board Members
+#### Membership
 
-Initially, a 2-year term is defined to expire on December 31, 2016 and
-a 1-year term is defined to expire on December 31, 2015.  Initial
-member of the board shall be (with term length in parentheses):
+* The Advisory Board shall consist of at least 3 members and no more than 5 members. The Advisory Board selects how many members within this range are needed.
+* All members shall serve a maximum of a two-year term, and may be re-appointed at the end of their term.
+* Membership terms should be staggered so that approximately half the number of members will end their terms in any given year.
+* The Advisory Board may appoint members to its number up to the maximum number of members.
+* The Advisory Board should work to ensure that the composition of the Advisory Board represents a diverse cross-section of the scientific community with an interest in SunPy. This diversity should consider things such as research topic, career stage, as well as all other forms of diversity.
```

Comment:
I think this is more relevant to the board as they self-elect members. The steering committee is elected by the custodians, so it would be hard to place a requirement like this on that. We could add something to the custodians in terms of  "encouraging nominations of people from a diverse section of the community", but really *anyone* who meets the requirements should be a custodian. The challenge in diversity there is encouraging the participation in the community in the first place?